src/shared/components/Header/Navigation/component.tsx

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
 
+import Button from 'shared/components/Button';
```

Comment:
let's import Button to `shared/component/index.ts` and then call it from there. like

```js
import Button from 'shared/components';
```

Reply:
Done!